### DISCUSSION TITLE
Clickhouse backup status query does not change from "in progress" to "success"

### DISCUSSION THREAD

**Discussion by nnor1** [Altinity/clickhouse-backup#1051] [Q&A] [answered]
Hello everyone,

We create our backups with the help of a CronJob that runs daily. When a job is created, it sends a POST request “backup/create?name=$BACKUP_NAME” to the container. The backup (metadata.json) is also created locally. A status query (backup/status) is also executed every minute in the job, but the status does not change from “in progress” to “success” from time to time as soon as the backup has been successfully created. As a result, no upload request can be executed afterwards.

For your information:
- The status change from “in progress” to “success” then occurs after approx. 2 hours. From then on, the backup can be uploaded again or a new one can be created.
- We use the image --> altinity/clickhouse-backup:2.6.2

Is this error known?

Greetings

**Comment by Slach** [comment, marked as answer]
> but the status does not change from “in progress” to “success” from time to time as soon as the backup has been successfully created

Check logs for clickhouse-backup container 
"in progess" means backup is not complete

look to https://github.com/Altinity/clickhouse-backup/blob/master/Examples.md#how-to-use-clickhouse-backup-in-kubernetes

for examples
